pkgs/os-specific/linux/xp-pen-drivers/deco-01-v2/default.nix

```
@@ -0,0 +1,64 @@
+{ lib, stdenv, fetchzip, libusb, glibc, libGL, xorg, qtx11extras, wrapQtAppsHook, autoPatchelfHook,
+libX11, libXtst, libXi, libXrandr, libXinerama, ... }:
+
+let
+  dataDir = "var/lib/xppend1v2";
```

IMO it'd be best to match the packages name:
```suggestion
  dataDir = "/var/lib/xp-pen-01-v2";
```
Unless that's inconsistent with other stuff in `/var/lib`? I'm not entirely sure.

---

That's not really possible, unfortunately, because the original data path is `/usr/lib/pentablet`, so to replace it in the binary it's only possible to pick paths of the same length or lower, and `xppend1v2` is the best I could do that matched the length of `pentablet`.

---

Fair enough 👍